AutoRest/Generators/CSharp/Azure.CSharp.Tests/project.json

```diff
@@ -24,14 +24,14 @@
   },
 
   "dependencies": {
-    "Microsoft.NETCore.App": { 
-      "type": "platform", 
-      "version": "1.0.0-rc2-*" 
+    "Microsoft.NETCore.App": {
+      "type": "platform",
+      "version": "1.0.0"
     }, 
-    "Microsoft.NETCore.Platforms": "1.0.1-rc2-24027",
+    "Microsoft.NETCore.Platforms": "1.0.1",
     "CSharp.Tests": "2.0.0",
-    "Microsoft.Rest.ClientRuntime.Azure": "[3.3.0,4.0.0)",
-    "xunit": "2.1.0",
-    "dotnet-test-xunit": "1.0.0-rc2-build10015"
+    "Microsoft.Rest.ClientRuntime.Azure": "[3.3.1,4.0.0)",
```

Comment:
FYI, for peer references like:
``` json
   "Microsoft.Rest.ClientRuntime.Azure": "[3.3.1,4.0.0)", 
```

I've been using:

``` json
   "Microsoft.Rest.ClientRuntime.Azure": {  "target": "project" } 
```
Which tells dotnet-cli to use the sibling project. 

I don't think you should change this, at this point, but I will check out what happens in the final package if you do it with `target:project`...